### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Sexuality/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Premise
-An analysis of the human sexuality oriented toward what society needs to accept and how. Religions always tried to define and impose a good sexual behavior. In the year 2016, how should society should regard sexuality?
-Our society right now is a comprehension of different cultures and religions, what should the law allow, what should be repressed?
-A wide analysis of the human behavior from the most sensual relationships, to the deepest kinks. From homosexuality to pedophilia.
+This is an analysis of human sexuality and how it is oriented towards what society needs to accept and why? Religions have always tried to define and impose a universally good sexual behavior. In 2016, how should society should regard sexuality?
```
Comment (Pella86) [inline Sexuality/README.md]: I think that universally is a bit much, religion tries to define sexual behavior, but some religions do take in account for regional preferences...

File: Sexuality/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #Premise
-An analysis of the human sexuality oriented toward what society needs to accept and how. Religions always tried to define and impose a good sexual behavior. In the year 2016, how should society should regard sexuality?
-Our society right now is a comprehension of different cultures and religions, what should the law allow, what should be repressed?
-A wide analysis of the human behavior from the most sensual relationships, to the deepest kinks. From homosexuality to pedophilia.
+This is an analysis of human sexuality and how it is oriented towards what society needs to accept and why? Religions have always tried to define and impose a universally good sexual behavior. In 2016, how should society should regard sexuality? 
+Today our society is a medley of different cultures and religions, so which beliefs should the law allow and which should be repressed?
```
Comment (Pella86) [inline Sexuality/README.md]: It's not about allowing beliefs, is about what legislation should be made...

File: Sexuality/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #Premise
-An analysis of the human sexuality oriented toward what society needs to accept and how. Religions always tried to define and impose a good sexual behavior. In the year 2016, how should society should regard sexuality?
-Our society right now is a comprehension of different cultures and religions, what should the law allow, what should be repressed?
-A wide analysis of the human behavior from the most sensual relationships, to the deepest kinks. From homosexuality to pedophilia.
+This is an analysis of human sexuality and how it is oriented towards what society needs to accept and why? Religions have always tried to define and impose a universally good sexual behavior. In 2016, how should society should regard sexuality? 
+Today our society is a medley of different cultures and religions, so which beliefs should the law allow and which should be repressed? 
+The section is a wide analysis of the human behavior from the most sensual relationships to the deepest kinks and of the taboo behaviors like homosexuality to pedophilia.
```
Comment (Pella86) [inline Sexuality/README.md]: The section will be... (is not yet complete)

Comment (Pella86) [inline Sexuality/README.md]: Homosexuality isn't taboo in many cultures...

File: Sexuality/Homosexuality.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
+Other animals have expressed homosexual behavior, but we lack records prior to 1970 because biologists 
+ignored such behaviors. 
+Lions use it for dominance assessing (same as the Roman empire), penguins being semelparous (like most birds) 
+engage homosexual behavior their entire life. 
 
 ##Personal choice or predisposition
-Homosexuality isn't a choice, like isn't a choice to be hetero. The sexual attraction, albeit can be guided, cannot be completely replaced. I would call it a predisposition. 
-Sexual orientation and sexual preference come both from our conscious and subconscious thoughts.
-There are gay reformatory, were people try to impose the heterosexuality. I have no idea about the success rate,
+Homosexuality isn't a choice, like it isn't a choice to be hetero. The sexual attraction, although it can be guided, it cannot be completely replaced. I would consider it a predisposition.
```
Comment (Pella86) [inline Sexuality/Homosexuality.md]: Call in the sense I would define it a predisposition. Considering is much more stronger...

File: Sexuality/Homosexuality.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
-Sexual orientation and sexual preference come both from our conscious and subconscious thoughts.
-There are gay reformatory, were people try to impose the heterosexuality. I have no idea about the success rate,
+Homosexuality isn't a choice, like it isn't a choice to be hetero. The sexual attraction, although it can be guided, it cannot be completely replaced. I would consider it a predisposition. 
+Sexual orientation and sexual preference come both from our conscious and subconscious thoughts. 
+There are gay reformatories, where people try to impose the heterosexuality. I have no idea about the success rate, 
 but I would say is very low.
 
 ##Minoritarian behavior
-One last thing, being gay will always be controversial in a population, since will always be an alternative behavior of the standard.
+One last thing, being gay will always be controversial in a population and itwill always be considered an alternative behavior of the standard heterosexuality.
```
Comment (Pella86) [inline Sexuality/Homosexuality.md]: typo 'itwill'

**Comment by Pella86** [comment]
I'm not sure how it works, but I reviewed your comments, now I think you should change, what I commented, and answer to me here...

**Comment by ghost** [comment]
Done.

**Review by Pella86 — APPROVED** [review]
Perfect! truly thank you for the corrections!!

**Pull request merged**